src/android/CodePush.java

```diff
@@ -2,6 +2,10 @@
 
 import android.content.pm.PackageManager;
 import android.os.AsyncTask;
+import android.util.Base64;
+
+import com.auth0.jwt.JWTVerifier;
```

Comment:
Probably we should use anither JWT library for decoding due to current library implementation consume a lot of methods. Please refer to this issue for details: https://github.com/Microsoft/react-native-code-push/issues/1002

Reply:
Sorry about that. I was following the react native implementation, but for some reason I was looking at old code. If it is OK for you I will try with the same library they have now:
`com.nimbusds:nimbus-jose-jwt:5.1`

Reply:
That should be ok, thanks for staying in touch!